src/hotspot/share/asm/codeBuffer.hpp

```diff
@@ -702,4 +706,19 @@ inline bool CodeSection::maybe_expand_to_ensure_remaining(csize_t amount) {
   return false;
 }
 
+inline int CodeSection::alignment(int section) {
+  if (section == CodeBuffer::SECT_CONSTS) {
+    return (int) sizeof(jdouble);
```

Comment:
This breaks Graal which puts data items larger than a `jdouble` (e.g.  32-byte vector masks) into the constants section.

Reply:
I opened https://bugs.openjdk.org/browse/JDK-8293999 to track this.